pkg/kubelet/dockershim/network/cni/cni_test.go

```diff
@@ -161,35 +167,14 @@ func TestCNIPlugin(t *testing.T) {
 	netName := fmt.Sprintf("test%d", rand.Intn(1000))
 	binName := fmt.Sprintf("test_vendor%d", rand.Intn(1000))
 
-	podIP := "10.0.0.2"
-	podIPOutput := fmt.Sprintf("4: eth0    inet %s/24 scope global dynamic eth0\\       valid_lft forever preferred_lft forever", podIP)
-	fakeCmds := []fakeexec.FakeCommandAction{
-		func(cmd string, args ...string) exec.Cmd {
-			return fakeexec.InitFakeCmd(&fakeexec.FakeCmd{
-				CombinedOutputScript: []fakeexec.FakeCombinedOutputAction{
-					func() ([]byte, error) {
-						return []byte(podIPOutput), nil
-					},
-				},
-			}, cmd, args...)
-		},
-	}
-
-	fexec := &fakeexec.FakeExec{
-		CommandScript: fakeCmds,
-		LookPathFunc: func(file string) (string, error) {
-			return fmt.Sprintf("/fake-bin/%s", file), nil
-		},
-	}
-
 	mockLoCNI := &mock_cni.MockCNI{}
 	// TODO mock for the test plugin too
 
 	tmpDir := utiltesting.MkTmpdirOrDie("cni-test")
 	testConfDir := path.Join(tmpDir, "etc", "cni", "net.d")
 	testBinDir := path.Join(tmpDir, "opt", "cni", "bin")
 	testDataDir := path.Join(tmpDir, "output")
-	defer tearDownPlugin(tmpDir)
+	//defer tearDownPlugin(tmpDir)
```

Comment:
This seems like a leftover?

Reply:
yes, I removed it to check cni-test results and invoke this test manually.